### ISSUE TITLE
Imports are mangled when using "ordered-imports" with prettier-vscode

### ISSUE THREAD

**Issue by liuweiGL** [azz/prettier-tslint#26]
### Bug Report

-  __VS Code version__: 1.30.1
- __Prettier - Code formatter version__: 1.7.3
- __TSLint version__:  5.11.0
- __TypeScript version__: 3.2.2
- __Running TSLint via__: VSCode 

#### TypeScript code being linted

```ts

import {NgModule} from '@angular/core'
import {AppComponent} from './app.component'
import {CommonModuleCustom} from './modules/common.module'
import {ElectronService} from './providers/electron.service'
import {LayoutComponent} from './components/layout/layout.component'

```

with `tslint.json` configuration:

```yml

  ordered-imports:
    - true
    - grouped-imports: false
      import-sources-order: case-insensitive
      named-imports-order: case-insensitive
      module-source-path: full
  # From tslint-eslint-rules
  object-curly-spacing: 
    - true
    - always

```

#### Actual behavior
![image](https://user-images.githubusercontent.com/21048878/50531933-792e2c80-0b4c-11e9-8ccf-c04e200e5728.png)

#### Expected behavior

![image](https://user-images.githubusercontent.com/21048878/50531956-c8745d00-0b4c-11e9-8944-66e3cab46b61.png)

#### Reproduce

You can reproduce from this repository: https://github.com/liuweiGL/prettier-tslint-bug-reproduce

#### PS

 I think the `apply-fixes.js` has some problems，the tslint‘s `applyTSLintAllFixes` function will update `Program`, but you don't.

Refrence from https://github.com/palantir/tslint/pull/2864

**Comment by aleclarson** [comment]
Duplicate of #19 

Thanks for the repro. Looking into a fix..

**Comment by liuweiGL** [comment]
> Duplicate of #19
> 
> Thanks for the repro. Looking into a fix..

Okay thank you.

**Comment by aleclarson** [comment]
Fixed in v0.4.2 🎉
